package.json

```diff
@@ -38,12 +39,14 @@
     "@typescript-eslint/eslint-plugin": "^7.13.1",
     "@typescript-eslint/parser": "^7.13.1",
     "@vitejs/plugin-react": "^4.3.1",
+    "autoprefixer": "^10.4.19",
     "eslint": "^8.57.0",
     "eslint-plugin-react-hooks": "^4.6.2",
     "eslint-plugin-react-refresh": "^0.4.7",
     "postcss": "^8.4.39",
     "postcss-preset-mantine": "^1.15.0",
     "postcss-simple-vars": "^7.0.1",
+    "tailwindcss": "^3.4.4",
```

Comment:
If you are using mantine, do you need tailwind?

If not, please remove it from the dependencies.

Reply:
Bhai I've created a ticket for UI update. I will remove it then. I'm actually using tailwindcss for manual styling of some elements.

Reply:
Okay no need to remove it then. Should be fine for this application.

But for future reference, it's better to stick to one ui library for a project especially considering mantine and tailwind follow different css paradigms (styled components and inline styling respectively).